devicetypes/smartthings/ezex-smart-occupancy-sensor.src/ezex-smart-occupancy-sensor.groovy

```diff
@@ -0,0 +1,106 @@
+        capability "Health Check"
+        capability "Sensor"
+       
+        fingerprint profileId: "0104", deviceId: "0107", inClusters: "0000, 0003, 0004, 0406", outClusters:"0006, 0019", manufacturer:"", model:"E280-KR0A0Z0-HA", deviceJoinName: "eZEX Smart Occupancy Sensor(AC Type)"
+    }
+    tiles(scale: 2) {
+        multiAttributeTile(name: "motion", type: "generic", width: 6, height: 4) {
+            tileAttribute("device.motion", key: "PRIMARY_CONTROL") {
+                attributeState "active", label: 'motion', icon: "st.motion.motion.active", backgroundColor: "#00A0DC"
+                attributeState "inactive", label: 'no motion', icon: "st.motion.motion.inactive", backgroundColor: "#cccccc"
+            }
+        }
+
+        standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+            state "default", action: "refresh.refresh", icon: "st.secondary.refresh"
+        }
+        main(["motion"])
+        details(["motion", "refresh"])
+    }
+}
+
+def installed(){
+    log.debug "installed"
+    return zigbee.readAttribute(0x0406, 0x0000)
+}
+
+def parse(String description) {
+    log.debug "description(): $description"
+    def map = zigbee.getEvent(description)
+    if (!map) {
+		if (description?.startsWith('read attr')) {
+            map = parseReportAttributeMessage(description)
+        } else {
+           log.debug "not handle"
+        }
+    }
+    log.debug "Parse returned $map"
+    def result = map ? createEvent(map) : [:]
+    if (description?.startsWith('enroll request')) {
+        List cmds = zigbee.enrollResponse()
+        log.debug "enroll response: ${cmds}"
+        result = cmds?.collect { new physicalgraph.device.HubAction(it) }
+    }
+    return result
+}
+
+private Map parseReportAttributeMessage(String description) {
+        Map descMap = (description - "read attr - ").split(",").inject([:]) { map, param ->
+                def nameAndValue = param.split(":")
+                map += [(nameAndValue[0].trim()):nameAndValue[1].trim()]
+        }
+        log.debug "Desc Map: $descMap"
+
+        Map resultMap = [:]
+        if (descMap.cluster == 0x0406 && descMap.attrId == 0x0000) {
+            log.debug "motion"
+            resultMap.name = "motion"
+            resultMap.value = descMap.value.endsWith("01") ? "active" : "inactive"
+        }
+
+        return resultMap
+}
+
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
+    log.debug "ping "
+    return zigbee.readAttribute(0x0406, 0x0000)
+}
+
+def refresh() {
+    log.debug "Refreshing Values"
+    return  zigbee.readAttribute(0x0406, 0x0000)
+}
+
+def configure() {
+    log.debug "configure"
+    //this device will send occupancy status every 5 minutes
+    sendEvent(name: "checkInterval", value:5 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
```

Comment:
Should we also not have an `enrollResponse()` here?

Reply:
As i know only security zone device need call enrollResponse(), this device is not IAS zone device, since not use cluster 0x0500.